packages/palette/src/elements/Typography/Typography.tsx

```diff
@@ -26,9 +26,10 @@ import {
   MaxWidthProps,
   space,
   SpaceProps,
-  style,
   textAlign,
   TextAlignProps,
+  verticalAlign,
+  VerticalAlignProps,
```

Comment:
`verticalAlign` is included in the new `layout` helper which we should use instead. 

> The layout utility includes style props for `width`, `height`, `display`, `minWidth`, `minHeight`, `maxWidth`, `maxHeight`, `size`, `verticalAlign`, `overflow`, `overflowX`, and `overflowY`.

https://styled-system.com/api#layout

Reply:
(Likewise, a lot of the other props can be replace with `typography` )

https://styled-system.com/api#typography

Reply:
So, I figured I'd update Typography on it's own because it can mostly go away with fixing the font-face declarations.